doc/tools.rst

```diff
@@ -162,7 +162,7 @@ Alternative Drivers
 These are alternatives to PyMongo.
 
 * `Motor <https://github.com/mongodb/motor>`_ is a full-featured, non-blocking
-  MongoDB driver for Python Tornado applications.
+  MongoDB driver for Python Tornado applications. Motor will be officially deprecated on May 14th, 2026.
```

Comment:
Could you mention why by saying that motor's features has been merged into pymongo via the new AsyncMongoClient api.